mayastor/src/subsys/mbus/registration.rs

```diff
@@ -166,10 +166,10 @@ impl Registration {
         // the message has been sent to the nats server, but we don't know
         // whether the control plane has received it or not
         // we could use request/reply to guarantee that it was delivered
-        debug!(
-            "Registered '{}' and grpc server {}",
-            self.config.node, self.config.grpc_endpoint
-        );
+        // debug!(
```

Comment:
this used to be sent every 30s/60s but now it's 5s so it's too many indeed, maybe it should be a trace.